package.json

```diff
@@ -15,8 +16,16 @@
     "hof-theme-govuk": "^4.5.0"
   },
   "devDependencies": {
-    "eslint": "^3.14.0",
-    "eslint-config-homeoffice": "^2.1.0"
+    "@types/chai": "^4.2.13",
+    "@types/mocha": "^8.0.3",
+    "@types/sinon": "^9.0.8",
+    "@types/sinon-chai": "^3.2.5",
```

Comment:
what are these and why do you need it?

Reply:
Some IDEs do not recognise variables and functions from these test libraries without these dependencies, just makes for a nicer developer experience to have them.